pkg/cosign/tuf/client.go

```diff
@@ -306,11 +314,9 @@ func NewFromEnv(ctx context.Context) (*TUF, error) {
 
 func Initialize(ctx context.Context, mirror string, root []byte) error {
```

Comment:
(this comment is for the return for NewFromEnv)

Shouldn't we be pulling the root from the cache if present? Otherwise, we're always fetching all roots starting with the root of trust. I'd hope there's some mechanism in TUF to ask for the latest cached root, and it verifies from a trusted root.

Reply:
Ah i had missed that line, looks good!